app/controllers/api/account.php

```diff
@@ -3683,7 +3683,7 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
     });
 
 App::get('/v1/account/mfa/factors')
-    ->desc('List Factors')
```

Comment:
what are all these lower case changes for?

they dont work good in avatars for example